contents/handbook/people/team-structure/product-analytics/objectives.mdx

```diff
@@ -1,72 +1,12 @@
-    1. **Visualizations**
-       * Improve breadth of visualization options (i.e. include stacked area graphs)
-         * Why? People are exporting data from PostHog to analyze in excel today they would easily churn from PostHog and rely entirely on other tools if we cannot offer them the visualizations they need 
-       * Improve the querying experience for Insights 
-         * Why? We have experimented a number of times with different configuration panels but none have been quite right, ship a new experience that makes querying easier to help new users keep coming back
-    2. **Notifications**
-        * Centralized alerting for actionable notifications 
-          * Why? Users expect to receive timely feedback of asynchronous and long running actions. Our system of bespoke emails and popup notifications will not scale to meet the multitude of new notification types. 
-        * Better notifications
-          * Some ideas what we can build with a centralised system:
-            * Receive results as soon as an experiment is significant 
-            * Set threshold on insights and notify if it’s crossed 
-            * Send notification for new recordings including a certain event 
-            * Introduce subscriptions to correlations 
-            * If something correlates really well with a key metric, notify me 
-            * Allow all of the above via email, slack, or webhook 
-            * Horizontal goal annotations with notifications as you move towards or away
-    3. **Onboarding**
-        * Improve first time user experience 
-          * Why? Some people don’t retain because they don’t fully understand how to use the product, they play around but haven’t truly activated, ensuring users can perform multiple meaningful discoveries will lead to more retained users 
-        * Remind people who’ve forgotten about PostHog to come back via email 
-          * Why? Some people like PostHog but have been moved to a different project for a few weeks and may forget about it, bringing them back will increase attention
-    4. **Upselling**
-        * Make the experience of upgrading to paid seamless and consistent 
-          * Why? Customers using our paid features have a very high retention rate, if we can get more customers subscribed to paid features this will increase our overall retention rate
-
-</HideFromJobPosting>
\ No newline at end of file
+- Objective: Get 80% of active b2b customers to add their marketing website
+  - Why? Increase # events coming in, which increases revenue. Our hypothesis is that marketing websites have 10-100x more events than applications.
+- Objective: Implement most frequently requested features with 3+ happy companies for the feature launch
+  - Key results:
+    - Implement the following:
+      - Multiple breakdowns
+      - Cohort style queries (did x event x times), but within a query itself
+      - Composite events: ‘Any event’, ‘New user’ (users first event), ‘top events’
```

Comment:
For results that expand the capability of insight building, how will we balance that with team west's SQL interface effort? Just thinking it could be complex to try to build an alternate interface for a feature that is also expanding in capability. Mostly asking so we can coordinate between teams working in the same area of the product.